src/main/msp/msp.c

```diff
@@ -2662,7 +2671,19 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
             // Added in MSP API 1.43
             currentPidProfile->motor_output_limit = sbufReadU8(src);
             currentPidProfile->auto_profile_cell_count = sbufReadU8(src);
+#if defined(USE_DYN_IDLE)
             currentPidProfile->idle_min_rpm = sbufReadU8(src);
+#else
+            sbufReadU8(src);
+#endif
+        }
+        if(sbufBytesRemaining(src) >= 1) {
```

Comment:
Space after `if` please.

Reply:
To keep the code consistent #ifdef without parentheses () should be more appropriate than #if defined() ?

Reply:
There is a mix if `#ifdef` and `#if defined()` in the code. I personally prefer the use of `#if defined()` to keep it consistent with more complex uses of `#if`.